mc_interface.c

```diff
@@ -2553,7 +2555,7 @@ static THD_FUNCTION(sample_send_thread, arg) {
 		}
 
 		for (int i = 0;i < len;i++) {
-			uint8_t buffer[40];
+			uint8_t buffer[39];  // 1 + 4 + 4 + 4 + 4 + 4 + 4 + 4 + 4 + 4 + 1 + 1
```

Comment:
I am confused why this value was 40. I counted the bytes and with the addition of 4 bytes it should now be 39. Could someone double-check?